db/db_compaction_test.cc

```diff
@@ -6464,82 +6458,366 @@ TEST_P(DBCompactionTestWithParam,
   }
 }
 
-INSTANTIATE_TEST_CASE_P(DBCompactionTestFIFOCheckConsistencyWithParam,
-                        DBCompactionTestFIFOCheckConsistencyWithParam,
-                        ::testing::Values("FindIntraL0Compaction",
-                                          "PickCompactionToWarm",
-                                          "CompactRange", "CompactFile"));
+class DBCompactionTestL0FilesReorderingCorruption : public DBCompactionTest {
+ public:
+  DBCompactionTestL0FilesReorderingCorruption() : DBCompactionTest() {}
+  void SetupOptions(const CompactionStyle compaciton_style,
+                    const std::string compaction_path_to_test = "") {
+    options_ = CurrentOptions();
+    options_.create_if_missing = true;
+    options_.compression = kNoCompression;
+
+    options_.force_consistency_checks = true;
+    options_.compaction_style = compaciton_style;
+    // TODO: expand test coverage to num_lvels > 1 for universal compacion,
```

Comment:
I wanted to expand test coverage to num_levels > 1 for universal compaction as mentioned in PR summary to cover some edge case in PickCompaction() but couldn't quite force the compaction to L0 - any good idea?